### PR TITLE
Fix section sorting issue on some devices with Simplified Chinese locale

### PR DESCRIPTION
**Pull request by junyuecao**

### CODE REVIEW

File: app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java [hunk 1]
```diff
@@ -311,7 +311,9 @@ private void onAppsUpdated() {
         // coalesce sections
         Locale curLocale = mLauncher.getResources().getConfiguration().locale;
         TreeMap<String, ArrayList<AppInfo>> sectionMap;
-        boolean localeRequiresSectionSorting = curLocale.equals(Locale.SIMPLIFIED_CHINESE);
+        boolean localeRequiresSectionSorting =
+                Locale.SIMPLIFIED_CHINESE.getLanguage().equals(curLocale.getLanguage())
```
Comment (nyancrimew) [inline app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java:315]: This equality should already be granted with the general equality check over Locale itself from what I can see. Could you please show an example of behavior before and after this patch and what the expected behavior is?

Comment (junyuecao) [reply app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java:315]: On some devices I think they are different and `equals` return `false`.

Comment (fonix232) [reply app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java:315]: I think in this case you should compare baseLocale instead, no?

Comment (junyuecao) [reply app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java:315]: BaseLocale and getBaseLocale() are not public.

Comment (junyuecao) [reply app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java:315]: Any update or better solution?

**Pull request merged**